### PR TITLE
IA-4607: Address IO blocking calls for GCE, Dataproc, and GKE

### PR DESCRIPTION
**Pull request by rtitle**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4607

## Dependencies
https://github.com/broadinstitute/workbench-libs/pull/1561/

### What

-

### Why

-

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.10.0"
 
-  private val workbenchLibsHash = "f31b537"
+  private val workbenchLibsHash = "8711e331-SNAP"
```
Comment (rtitle) [inline project/Dependencies.scala]: TODO: update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/1561/ is merged

**Comment by rtitle** [comment]
jenkins retest

**Review by LizBaldo — APPROVED** [review]

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Pull request merged**